code/game/objects/items/circuitboards/machine_circuitboards.dm

```diff
@@ -757,6 +757,15 @@
 		/obj/item/reagent_containers/glass/beaker = 2,
 		/obj/item/stack/sheet/glass = 1)
 
+/obj/item/circuitboard/machine/limbgrower/fullupgrade
+	name = "Limb Grower (Machine Board)"
+	greyscale_colors = CIRCUIT_COLOR_MEDICAL
+	build_path = /obj/machinery/limbgrower
```

Comment:
the full upgrade machine drops this chip, but this chip builds the normal machine?

Reply:
Tried to discourage people from getting the board, if they try to do something dumb to get the board and build a limb grower, it'll just end up as a basic one.

Reply:
okay, i'm alright with that

Reply:
why not just make it drop the normal circuit board then

Reply:
make it build the full upgrade limb grower. You are already requiring femto and bluespace glass beakers so they won't be able to complete it without those components